src/components/airqualityindex/DataDisplay.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DataDisplay = ({ name, value }) => (
+  <div className="detail-data flex">
+    <p className="data-name">
+      {/* eslint-disable-next-line react/no-danger */}
```

Comment:
**[OPTIONAL]**

- Hey, The `dangerouslySetInnerHTML` property is used to set HTML directly from React. It is called "dangerous" because it can be used to inject malicious scripts into your page if not used properly. 
 it is recommended to avoid using `dangerouslySetInnerHTML` and instead use React's built-in methods for rendering content. If you must use it, make sure to sanitize any user-generated content before rendering it with `dangerouslySetInnerHTML`.